sdk/ai/azure-ai-inference/samples/sample_chat_completions_from_input_prompt_string.py

```diff
@@ -0,0 +1,79 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to get a chat completions response from
+    the service using a synchronous client, and directly providing the 
+    input in string format.
+
```

Comment:
Please update description to mention that this sample uses Prompty to manage messages

Reply:
I would also add a link to the Prompty website

Reply:
Actually, I heard different answers for this... one way to think about it is, we are providing "prompt asset" but leverage Prompty as internal implementation; the other way is we are providing prompt asset from Prompty. The current comment suggested the first one, but I'm open to any suggestions

Reply:
Update: confirmed from PMs, we focus on "prompt" instead of "Prompty", in other words, Prompty is the internal implementation. Updated the comment with the decision